### ISSUE TITLE
The new 3.1.6 dll is test-signed

### ISSUE THREAD

**Issue by TMUNYU** [dotnet/SqlClient#2793]
### Describe the bug
We are getting an error after upgrading to this new dll (Microsoft.Data.SqlClient 3.1.6). 

```
ERROR: System.IO.FileLoadException: Could not load file or assembly 'Microsoft.Data.SqlClient, Version=3.0.0.0, Culture=neutral, PublicKeyToken=23ec7fc2d6eaa4a5' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A)
13:55:20     File name: 'Microsoft.Data.SqlClient, Version=3.0.0.0, Culture=neutral, PublicKeyToken=23ec7fc2d6eaa4a5' ---> System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
```

### To reproduce
Getting the error every time we try to load the dll in an app domain.

On further investigation, we found that `sn.exe -v Microsoft.Data.SqlClient.dll` is giving us `Microsoft.Data.SqlClient.dll is a delay-signed or test-signed assembly` instead of `Microsoft.Data.SqlClient.dll' is valid`

### Expected behavior
sn.exe should return valid

### Further technical details
Microsoft.Data.SqlClient version: 3.1.6
.NET target: Framework 4.7.2 and Standard 2.0
SQL Server version: SQL Server 2022
Operating system: Windows Server 2019 Standard

**Additional context**
none

**Comment by JRahnama** [comment]
@TMUNYU there has been a glitch in our release pipeline. The issue is fixed and 3.1.7 will be released shortly. Thank you for bringing up this to our attention.

**Comment by TMUNYU** [comment]
All good now. 3.1.7's signature is valid
